velox/exec/Spiller.h

```diff
@@ -37,8 +37,9 @@ class Spiller {
     kHashJoinProbe = 3,
     // Used for order by.
     kOrderBy = 4,
+    // Num of spiller types
```

Comment:
nit: Number of ...

Reply:
Fixed, and add `.` at the end of this comment line.